src/lib/stores/schema.ts

```diff
@@ -17,6 +18,7 @@ export interface CurrentBackend {
 export interface Configuration {
 	backends: BackendConfiguration[];
 	backend: CurrentBackend;
+	//// factories: { [factoryName: string]: (configuration: BackendConfiguration, model: string) => Backend }
```

Comment:
Please don't leave commented out parts like this around.

Reply:
Oh, good catch, thanks